### DISCUSSION TITLE
Can't update pages after upgrading 2.x to 3.x

### DISCUSSION THREAD

**Discussion by sjsiam** [area17/twill#2543] [Q&A]
I have recently upgraded from Twill CMS 2.X to 3.X. Everything works well except updating or publishing posts. I am getting this error:

`{message: 'Page', exception: 'A17\\Twill\\Exceptions\\NoCapsuleFoundException', file: 'G:\\fileion.com\\vendor\\area17\\twill\\src\\TwillCapsules.php', line: 83, trace: Array(65)}`

How can I fix it? Please help.

**Comment by adam-jones-net** [comment]
> I have recently upgraded from Twill CMS 2.X to 3.X. Everything works well except updating or publishing posts. I am getting this error:
> 
> `{message: 'Page', exception: 'A17\\Twill\\Exceptions\\NoCapsuleFoundException', file: 'G:\\fileion.com\\vendor\\area17\\twill\\src\\TwillCapsules.php', line: 83, trace: Array(65)}`
> 
> How can I fix it? Please help.

Did you get anywhere with this @siam02  ?  I have the same issue with a site I'm trying to upgrade.
